solvers/julia_fista.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
```

Comment:
@TheoGuyard could you make this an option of `julia_pgd` to reduce code duplication and mimick what's done in python_pgd?

Reply:
Yep, I'm on it 👍